README.md

```diff
@@ -1,186 +1,246 @@
+## Backers and sponsors
 
-poliastro comes from Polimi, which is the shortened name of the
-Politecnico di Milano, the Italian university where I was studying while
-writing this software. It's my tiny tribute to a place I came to love.
-*Grazie mille!*
+poliastro requires finnacial support to mantain its high quality standars. The
+money is used to renew the web domain and updating the documentation hosting
+subscription among others.
 
-## Is poliastro validated?
+If you would like to support poliastro, consider [becoming a backer] or
+[becoming a sponsor]. 
 
-Yes! poliastro is a community project that strives to be easy to use,
-while at the same time producing correct results
-[that are validated](https://github.com/poliastro/validation/)
-against other [commonly used Astrodynamics software](https://docs.poliastro.space/en/latest/related.html)
-such as GMAT and Orekit.
+**Thanks to all our backers!**
 
-## Can I suggest new features for poliastro?
+[![Backers](https://opencollective.com/poliastro/backers.svg?width=890)](https://opencollective.com/poliastro#backer)
 
-Sure, we encourage you to [open an issue](https://github.com/poliastro/validation/issues/new)
-so we can discuss future feature additions!
 
-## What's the future of the project?
+**Thanks to all our sponsors!**
 
-poliastro is actively maintained and receiving an influx of new
-contributors thanks to the generous sponsorship of Google and the
-European Space Agency. The best way to get an idea of the roadmap is to
-see the [Milestones](https://github.com/poliastro/poliastro/milestones)
-of the project.
+[![Sponsors](https://opencollective.com/poliastro/sponsor/0/avatar.svg)](https://opencollective.com/poliastro/sponsor/0/website)
+[![Sponsors](https://opencollective.com/poliastro/sponsor/1/avatar.svg)](https://opencollective.com/poliastro/sponsor/0/website)
+
+
+
+
+## Frequently asked questions
+
+* **What's up with the name?**
+
+  poliastro comes from Polimi, which is the shortened name of the
+  Politecnico di Milano, the Italian university where I was studying while
+  writing this software. It's my tiny tribute to a place I came to love.
+  *Grazie mille!*
+
+* **Is poliastro validated?**
+
+  Yes! poliastro is a community project that strives to be easy to use, while at
+  the same time producing correct results [that are validated] against other
+  [commonly used Astrodynamics software] such as GMAT and Orekit.
+
+* **Can I suggest new features for poliastro?**
+
+  Sure, we encourage you to [open an issue] so we can discuss future feature
+  additions!
+
+* **What's the future of the project?**
+
+  poliastro is actively maintained and receiving an influx of new
+  contributors thanks to the generous sponsorship of Google and the
+  European Space Agency. The best way to get an idea of the roadmap is to
+  see the [milestones] of the project.
+
+
+<!-- LINKS AND REFERENCES -->
+
+[quickstart guide]: https://docs.poliastro.space/en/latest/quickstart.html
```

Comment:
Note that people might arrive to the `latest` docs and try APIs that are not released yet. Maybe something like https://github.com/humitos/sphinx-version-warning could be shown in that case.